SalesforceCommerceCloud/pwa-kit
# [V2] Add favicon for `template-retail-react-app`

Currently, the icon favicon is not loading in `template-retail-react-app` in V2. We are seeing a `file not found` warning in the terminal.

Updated serve favicon flow to use `serveStaticFile` to maintain consistency with serving other static assets.
Removed `favIconPath` option from ssr config and removed related code to serve favicon in favor of using `serveStaticFile`

# Types of Changes


> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Run npm start inside `packages/template-retail-react-app`
- Verify terminal does not show a Favicon not found error
- Verify if browser loads correct favicon (Clearing cache and hard-reloading maybe required)

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
